scenes/game/local_game.gd

```diff
@@ -3156,12 +3450,19 @@ func do_hit_response_effects(offense_player : Player, defense_player : Player, i
 	# If more of these are added, need to sequence them to ensure all handled correctly.
 	var events = []
 	active_strike.strike_state = next_state
-	if not offense_player.strike_stat_boosts.ignore_armor and defense_player.strike_stat_boosts.when_hit_force_for_armor:
-		change_game_state(Enums.GameState.GameState_PlayerDecision)
-		decision_info.player = defense_player.my_id
-		decision_info.type = Enums.DecisionType.DecisionType_ForceForArmor
-		decision_info.choice_card_id = active_strike.get_player_card(defense_player).id
-		events += [create_event(Enums.EventType.EventType_Strike_ForceForArmor, defense_player.my_id, incoming_damage)]
+	if not offense_player.strike_stat_boosts.ignore_armor:
+		# Currently assumes these will be armor-related; probably breaks if choices get involved
```

Comment:
almost certainly breaks if choices get involved and block is also involved.
If someone has multiple when_hit that are choices or need to be ordered and interact with block, we'll have to add the same kind of effect list stuff as other timing windows.
For now, this should be fine though.